### PR TITLE
Update docu

### PR DESCRIPTION
**Pull request by schymans**
I added another jupyter notebook with all importable variables and equations listed neatly in tables and updated the api_examples notebook. I also integrated both of them in the documentation so that they can actually be easily found.

### CODE REVIEW

File: requirements-devel.txt [hunk 1]
```diff
@@ -1 +1,8 @@
 -e git+https://github.com/sympy/sympy.git#egg=sympy
+jupyter
+nbconvert
+pandoc
+nbsphinx
+IPython
+ipykernel
+nbsphinx
```
Comment (jirikuncar) [inline requirements-devel.txt]: you can include them as `extra_require = {'dev': ...}` in `setup.py`

**Comment by jirikuncar** [comment]
@schymans to fix travis you need to include `docs/_build` in `.gitignore`

File: .gitignore [hunk 2]
```diff
@@ -72,3 +72,10 @@ src/*
 
 # version.py
 essm/version.py
+
+# Jupyter temp files
+.ipynb_checkpoints
+*/.ipynb_checkpoints/*
+
+# Docu build
+docs/_build
```
Comment (jirikuncar) [inline .gitignore]:
```suggestion
# Documentation build
docs/_build/
```

**Comment by schymans** [comment]
I just found someone else with a similar problem: https://github.com/pypa/warehouse/pull/355/commits/c299d665392ed5245732c0d10d03141fc3a2455b
Now added exclude of `docs/_build` to MANIFEST.im, following that example.

File: setup.py [hunk 3]
```diff
@@ -40,13 +40,21 @@
...
+        'Sphinx==1.8',
         'matplotlib>=1.5.1',
         'sphinxcontrib-bibtex>=0.3.5',
+        'nbsphinx>=0.6.1'
     ],
     'generator': ['yapf>=0.16.2', ],
     'tests':
         tests_require,
+    'dev': ['jupyter',
+        'nbconvert',
```
Comment (jirikuncar) [inline setup.py]:
```suggestion
    'dev': [
        'jupyter',
        'nbconvert',
```

File: docs/conf.py [hunk 4]
```diff
@@ -140,7 +141,8 @@ def get_attr(obj, value, *args, **kwargs):
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', '_build',
```
Comment (jirikuncar) [inline docs/conf.py]:
```suggestion
exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store',
```

**Review by jirikuncar — APPROVED** [review]

**Pull request merged**